src/main/java/org/jabref/gui/groups/GroupDialog.java

```diff
@@ -268,11 +275,11 @@ public GroupDialog(JabRefFrame jabrefFrame, AbstractGroup editedGroup) {
                     if (explicitRadioButton.isSelected()) {
                         Character keywordDelimiter = Globals.prefs.getKeywordDelimiter();
                         if (groupName.contains(Character.toString(keywordDelimiter))) {
-                            jabrefFrame.showMessage(
+                            JOptionPane.showMessageDialog(null,
```

Comment:
We have a wrapper class (`DialogService`) to show notification/warning dialogs. Please use this one here and in the other places. Thanks

Reply:
Thank you. I will make the required changes and push the code.